tests/unit_tests/db_engine_specs/test_presto.py

```diff
@@ -106,3 +108,38 @@ def test_get_schema_from_engine_params() -> None:
         )
         is None
     )
+
+
+@mock.patch("superset.db_engine_specs.presto.PrestoEngineSpec.latest_partition")
+@pytest.mark.parametrize(
+    ["column_type", "column_value", "expected_value"],
+    [
+        (types.DATE(), "2023-05-01", "DATE '2023-05-01'"),
+        (types.TIMESTAMP(), "2023-05-01", "TIMESTAMP '2023-05-01'"),
+        (types.VARCHAR(), "abcdef", "'abcdef'"),
```

Comment:
For this example the string ‘2023-05-01’ is likely preferred given we’re talking about latest in a temporal sense.

Reply:
Updated